docs/releases/2.1.txt

```diff
@@ -180,69 +180,56 @@ Models
 
 * Query expressions can now be negated using a minus sign.
 
-* The new :class:`~django.db.models.functions.Abs` database function returns
-  the absolute value of an expression.
+* The new math database function Abs returns the absolute value of an
```

Comment:
What I meant is to have only 1 bullet point and link "math database functions" to the appropriate section. Don't mention every function and what it does in the release notes.

Reply:
I see. That's a good point. I have modified the doc accordingly. Now it starts to build after my most recent commit. Thank you!